components/EmojiPopover/EmojiPopover.module.scss

```diff
@@ -3,10 +3,22 @@
   &.light {
     background-color: var(--brightGray);
     border: 1px solid var(--lightGray);
+    .emojiItem {
+      color: var(--gray);
+      &:hover {
+        background-color: var(--lightGray);
+      }
+    }
   }
   &.dark {
     border: 0.5px solid var(--gray);
     background-color: var(--lightGray);
+    .emojiItem {
+      color: #fff;
+      &:hover {
+        background-color: var(--lightGray);
```

Comment:
this style is duplicated in light selector
background-color: var(--lightGray); -> it can be seen in line 9